### DISCUSSION TITLE
NO ME DEJA CREAR CUENTA

### DISCUSSION THREAD

**Discussion by Oscarcito122** [vercel/community#4059] [Account Issue] [answered]
### Question

estoy desde ayer intentando ingresar por primera vez a Vercel y siempre me tira error. Intente con mi github incluso con correos y nunca me deja. a alguien le paso y sabe como arreglarlo?

**Comment by amyegan** [comment, marked as answer]
Hey @Oscarcito122. Sorry you're having trouble creating an account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will get you in contact with the team that can help with login issues.
